### PR TITLE
Add CVE-2023-0461 mitigation to 2.5.9

### PR DESCRIPTION
**Pull request by keopp-hpe**
#### Summary and Scope
This PR adds the CVE-2023-0461 mitigation to the UAN 2.5.9 documentation.

NOTE: This procedure must be testing before merging to release/uan-2.5.9.

### CODE REVIEW

File: docs/portal/developer-portal/operations/CVE_2024_0461.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+The mitigation script provided below will perform the following actions:
+
+1. Select the first `BONDING_SLAVE0` from `ifcfg-bond0` if it exists
+1. Replace `bond0` in `ifcfg-can0` with the `BONDING_SLAVE0` interface
+1. Reload interfaces
+1. Add a blacklist for the tls module
+1. Unload `bonding` and `tls` kernel modules provided `mlx5_core` is not present
+1. Fail if `mlx5_core` is detected to highlight that the mititagtion failed
+
+*Important*: This mitigation is intend for UANs that meet the following criteria:
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/operations/CVE_2024_0461.md:26]: Do we want "Important" in italics or bold? I thought it was in bold in the other PRs.

**Review by darrenn-jackson — APPROVED** [review]
1 small question about formatting. Content itself looks good.

File: docs/portal/developer-portal/operations/CVE_2024_0461.md (hunk 1 above)
Comment (keopp-hpe) [reply docs/portal/developer-portal/operations/CVE_2024_0461.md]: Fixed to bold.

**Review by alanm-hpe — APPROVED** [review]

**Comment by keopp-hpe** [comment]
UAN 2.5.9 will not release.  Closing this PR.

**Pull request closed without merging**